rmqsrc/spec.nim

```diff
@@ -33,3 +36,29 @@ const
 
   FRAME_MAX_SIZE* = 131072
   BODY_MAX_LENGTH* = FRAME_MAX_SIZE - FRAME_HEADER_SIZE - FRAME_END_SIZE
+
+let
```

Comment:
We can't define `METHODS` and its reverse lookup counterpart `METHOD_VALS` as consts if we want to define one from the other. Kind of a bummer...